src/Sylius/Bundle/AttributeBundle/Form/EventSubscriber/BuildAttributeValueFormSubscriber.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Sylius\Bundle\AttributeBundle\Form\EventSubscriber;
 
+use Sylius\Bundle\AttributeBundle\Form\Type\AttributeValueType\Configuration\Factory\AttributeValueTypeConfigurationFactoryInterface;
```

Comment:
It's a little verbose indeed :D I warmly accept suggestion for renaming it. I put it there because it is a configuration to Attribute value type